rlfit.m

```diff
@@ -13,6 +14,9 @@
 % lb and ub are vectors of upper and lower bounds on parameters
 %
 % niter (optional) is the number of random restarts to use in fitting
+%
+% ispresentx is a vector, one per trial, indicating options present
```

Comment:
I think it would clarify things if you noted that it's Noptions x Ntrials (since vectors in Matlab can be row or column).

Reply:
Good call, thank you. I changed the description on that one.